lib/bundlex/project/precompiler.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Bundlex.Project.Precompiler do
```

Comment:
Wisdom of BBB: Where does the name `Precompiler` come from? To me it seems more like `Preprocessor` or even `Processor`

Reply:
I started from the processor, but it didn't sound well, so switched to precompiler. Agree that preprocessor would be better, but not sure if I worth changing